Source/Modules/ModuleCoPFollowTransform.cs

```diff
@@ -15,20 +15,23 @@ public class ModuleCoPFollowTransform : PartModule
         [KSPField]
         public string transformName;
 
+        [SerializeField]
         private Transform followTransform;
 
         public override void OnLoad(ConfigNode node)
         {
-            base.OnLoad(node);
-            if (HighLogic.LoadedScene != GameScenes.LOADING)
+            if (followTransform == null || followTransform.name != transformName)
             {
                 if (transformName != null) followTransform = part.FindModelTransform(transformName);
                 if (followTransform == null) Debug.LogError($"[{MODULENAME}] transformName was empty or does not exist.");
```

Comment:
would be a good idea to include the transform name that was searched for in the error message.